package.json

```diff
@@ -61,13 +61,13 @@
     "i18next-http-backend": "^1.1.0",
     "inversify": "^5.0.5",
     "lodash": "^4.17.21",
+    "maildev": "^1.1.0",
```

Comment:
I don't think we need this at all.. you're running maildev out of docker-compose, right?